web/templates/user_card_popover_content.hbs

```diff
@@ -1,172 +1,175 @@
 {{! Contents of the user card popover }}
-<ul class="nav nav-list user-card-popover-actions" id="user_card_popover" data-user-id="{{user_id}}">
-    <li class="popover_user_name_row">
-        <b class="user_full_name" data-tippy-content="{{user_full_name}}">{{user_full_name}}</b>
-        {{#if is_active }}
-            {{#if is_bot}}
-            <i class="zulip-icon zulip-icon-bot" aria-hidden="true"></i>
-            {{else}}
-            <span class="{{user_circle_class}} user_circle popover_user_presence tippy-zulip-tooltip hidden-for-spectators" data-tippy-content="{{user_last_seen_time_status}}"></span>
+<div class="popover-title"></div>
+<div class="popover-content">
```

Comment:
I think we need to get rid of these classes, which would also let you eliminate reindenting this file.

`git grep popover-content` shows that these classes are defined by `web/third/bootstrap-tooltip/tooltip.css`; they should not be used in a tippy popover as we plan to delete that whole file as soon as this popover migration is completed.

It looks like a few of the other popovers you've migrated need small tweaks related to this -- they should be using the equivalent tippy outer container class.

```
$ git grep popover-content
web/src/emoji_picker.js:                .find(".popover-content")
web/src/giphy.js:        // `popover-content`, so we need to reinitialize
web/styles/dark_theme.css:    .popover-content {
web/styles/dark_theme.css:    .hotspot.overlay .hotspot-popover .hotspot-popover-content,
web/styles/hotspots.css:    .hotspot-popover-content {
web/styles/popovers.css:.popover-content {
web/styles/popovers.css:    & .popover-content ul.nav {
web/styles/popovers.css:    & .popover-content ul.nav {
web/styles/popovers.css:    .popover-content {
web/styles/reactions.css:    .popover-content {
web/templates/emoji_popover.hbs:    <div class="popover-content"></div>
web/templates/giphy_picker.hbs:        <div class="popover-content" data-simplebar>
web/templates/hotspot_overlay.hbs:        <div class="hotspot-popover-content">
web/templates/no_arrow_popover.hbs:        <div class="popover-content">
web/templates/user_card_popover_content.hbs:<div class="popover-content">
web/templates/user_group_info_popover.hbs:        <div class="popover-content">
```

Reply:
I tried to get this done fast, but it looks like it needs its own PR. There's a lot to change and do. I think we can open some follow-up issue for that.

Reply:
Created #26821.